rush.json

```diff
@@ -138,6 +138,12 @@
       ]
     },
     {
+      "packageName": "@microsoft/api-documenter",
+      "projectFolder": "libraries/api-documenter",
+      "reviewCategory": "libraries",
+      "shouldPublish": false  // not published; this is a code sample for api-extractor
```

Comment:
I removed the comment.

If you want to fix all the shouldPublish=false in this file, feel free.  However I will point out that you're about to remove this from the file format.  :-)

---
In reply to: [134341985](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134341985) [](ancestors = 134341985)